app/static/js/trackVolunteers.js

```diff
@@ -68,6 +69,7 @@ $(document).ready(function() {
   });
 
   $(".removeVolunteer").on("click", function() {
+    $(".removeVolunteer").prop("disabled", true)
```

Comment:
Make sure if there is an error and the page doesn't reload that we can click the button again

Reply:
resolved